starfish/pipeline/filter/__init__.py

```diff
@@ -0,0 +1,49 @@
+from starfish.io import Stack
+from starfish.pipeline.pipelinecomponent import PipelineComponent
+from starfish.util.argparse import FsExistsType
+from . import _base
+from . import gaussian_low_pass
+from . import white_tophat
+
+
+class Filter(PipelineComponent):
+
+    filter_group = None
```

Comment:
I wonder if it's possible to type this.

Reply:
Yes, `argparse.ArgumentParser` apparently. Will add argparse types in future PR.